### ISSUE TITLE
Terminal font is crisp on the top, but gets blurry by middle

### ISSUE THREAD

**Issue by weiss-d** [Eugeny/terminus#2726]
Hi and thanks for this cool app! 
I have a problem though:
terminal font is crisp on the top, but gets blurry by middle (please see screenshot).

![image](https://user-images.githubusercontent.com/62437174/85314073-9a61a600-b4c1-11ea-84ed-5340d36ab3dd.png)

Win 10 with latest updates.

---- automatic info ----
Version: 1.0.112
Platform: win32 10.0.19041
Plugins: none

**Comment by jpbarraca** [comment]
Confirm that fonts look blurry in v1.0.112. I don't think it's just on the center of the terminal.

**Comment by THX1188** [comment]
Did you check on windows 10 that your sacleout (desktop > right-click > display settings [Scale and layout]) on your display is set to 100%; I had it 125% due to the high-res display which made the fonts looking more blurry,...

**Comment by weiss-d** [comment]
@THX1188 yes, I keep the scale setting on 125% because otherwise controls in all apps become very small  and inconvenient to use on my 1920x1080 display. Very few of all apps have a blurring issue due to this setting, and they all are blurred completely, while here only console text is blurred.

**Comment by THX1188** [comment]
I have a 3440 x 1440 Display and it gets even worse. I did two things to get a better visual experience.
- I set the Systemfonts larger with the Advanced System Font Tool 

![image](https://user-images.githubusercontent.com/61349287/86235118-3a8d8e00-bb98-11ea-9aec-1208b3c999ca.png)

- and with Ease of Access [Display] i set the font-size to 130%
![image](https://user-images.githubusercontent.com/61349287/86235348-97894400-bb98-11ea-97ba-634f55dfebc7.png)

That worked for me , the fonts now are crystal clear.

**Comment by kkrauth** [comment]
Same problem here when scaling is set to over 100%. Using a 3440x1440 display with 125%. Would love to see this one addressed.

P.S. I use "FiraCode Light" font which looks the best out of about 30 different fonts I tried, but it's still much blurrier than it should be.

**Comment by shaddu38** [comment]
Who have similar problem, you can try old version with "hterm" setting.
Tabby version 1.0.151
Settings->Terminal->Frontend: hterm
Restart Tabby
There i don't have this issue.

**Comment by trossr32** [comment]
Having the same issue with Windows scaling set to 100%
